api/conf/urls/__init__.py

```diff
@@ -35,6 +35,13 @@
     path("admin/", admin.site.urls),
     path("healthcheck/", HealthCheck.as_view(), name="health"),
     path("v1/", include(versioned_paths)),
+    path(
+        "robots.txt/",
```

Comment:
I don't know if there should be a trailing slash for a `.txt` file. I'm leaning towards no, there shouldn't be one.

```suggestion
        "robots.txt",
```

Reply:
ok will keep that in mind